docs/users/FAQ.md

```diff
@@ -46,21 +52,19 @@ Go to our [membership page](https://synshop.org/joining-syn-shop) and follow the
 
 ## What are the dues?
 
-Being part of the community is free as is joining our our  [Discord Server]( https://synshop.org/discord). It's also
-free to come down to  the shop and acclimate yourself when ever we're open. Meet our people, 
- and join in the conversation!
+Being part of the community is free as is joining our our [Discord Server](https://synshop.org/discord). It's also
+free to come down to  the shop and acclimate yourself when ever we're open. Meet our people, and join in the conversation!
 
-Using the equipment requires you to be a paying member. The dues are set at $50 per month. 
+Using the equipment requires you to be a paying member. Membership dues are presently $60 per month. 
 For this price, you gain access to all of our equipment during open hours. If you eventually become a 
-Vetted Member in good standing, you will have access to the shop and its equipment 24 hours a day 
+Vetted Member in good standing, you will have access to the shop and its equipment 24 hours a day, 
 7 days a week.
 
 ## How to I get involved?
 
-The best way to get involved with the group is to show up when we're open nights, join the 
- [Discord Server]( https://synshop.org/discord), 
-and just participate in
-things. People at SYN Shop have all been very open to hearing what other people are working on, 
+The best way to get involved with the group is to come visit the shop during the opening hours, join the 
+ mailing lists and [Discord Server](https://synshop.org/discord), 
```

Comment:
I think we're not playing up the mailing list, so I vote we not add this in.  

@jeddaniels  or @krux702  - thoughts?

Reply:
The mailing lists are dead because they're not easy to find.  I think users should be given the option, and they can use what they like.  As has been previously discussed, use of Discord requires agreement to an abusive legal contract with the company that operates Discord, in addition to giving up one's civil rights, and alternatives for participation that do not require that should be preserved on equal footing so that users can *use what they want*.

Everyone has an email, and email client. Not everyone has (or wants) a Discord account (and associated legal obligations to Discord Inc).

Regardless, I don't think this is the place for this discussion.  The email lists are in operation, and are linked to from the existing sites, and *are in active use by shop leadership* - I got an email via the members list from Krux on 5 September of this year.

Reply:
> I don't think this is the place for this discussion.

Agreed!  Please revert and we can discuss on Discord.